### PR TITLE
added modman config

### PR DESCRIPTION
**Pull request by mystix**
added modman config file

for easy installation using @colinmollenhour's excellent modman script

### CODE REVIEW

File: modman [hunk 1]
```diff
@@ -0,0 +1,8 @@
+src/app/code/community/*                                app/code/community/
+src/app/design/adminhtml/default/default/layout/*       app/design/adminhtml/default/default/layout/
+src/app/design/adminhtml/default/default/template/*     app/design/adminhtml/default/default/template/
+src/app/design/frontend/base/default/layout/*           app/design/frontend/base/default/layout/
+src/app/etc/modules/*                                   app/etc/modules/
+src/app/locale/en_US/*                                  src/app/locale/en_US/
+src/app/locale/zh_TW/*                                  src/app/locale/zh_TW/
```
Comment (jwswj) [inline modman]: Are the rest of the locales also required? I'm familiar with modman.
cc @mreeves1

Comment (mystix) [inline modman]: Ooops. My bad. Added all missing locales in my next commit.

**Comment by mreeves1** [comment]
Yes, (belatedly) I think that makes sense. Sorry I don't usually use locales and was in a rush to use this extension with a magento install using modman. Also how did you see my contribution? I thought I had issued a pull request to  zendesk with this and one other thing but I don't see it now.

**Comment by jwswj** [comment]
@mystix Thanks - looks like modman has a shortcut for if we need anymore translations in the future - https://github.com/colinmollenhour/modman/wiki/Productivity-Hints

@mreeves1 I've closed your old pull request https://github.com/zendesk/magento_extension/pull/10 as new PRs replaced it.

**Comment by jwswj** [comment]
:+1:

**Pull request merged**